.github/workflows/auto-merge.yml

```diff
@@ -42,13 +42,13 @@ jobs:
           git config user.email "70000568+nvauto@users.noreply.github.com "
           git fetch origin ${HEAD} ${BASE}
           git checkout -b ${INTERMEDIATE_HEAD} origin/${HEAD}
-          OUT=$(git --no-pager diff --name-only origin/${BASE} | grep "${FILE_USE_BASE}" || true)
-          [[ ! -z "${OUT}" ]] && git checkout origin/${BASE} -- ${FILE_USE_BASE} && \
-            git commit -s -am "Auto-merge use submodule in BASE ref"
+          git checkout origin/${BASE} -- ${FILES_USE_BASE}
+          [ -z "$(git status --porcelain=v1 --untracked=no) ] && \
```

Comment:
should be "$(git status --porcelain=v1 --untracked=no)"  ?

Reply:
Auto merge code change : https://github.com/NvTimLiu/spark-rapids-jni/pull/73

Not merge submodule change: https://github.com/NvTimLiu/spark-rapids-jni/pull/75 (nothing changed on branch-24.06)

Reply:
Thanks!